data/campaigns/Son_Of_The_Black_Eye/scenarios/03_Toward_Mountains_of_Haag.cfg

```diff
@@ -442,7 +442,7 @@
             [else]
                 [message]
                     speaker="Kapou'e"
-                    message= _ "Grüü, many thanks for your help in this fight. I’m sorry for the loss of your father. Would you like to join us in our journey? Your help would be useful to me."
+                    message= _ "Grüü, many thanks for your help in this fight. I’m sorry for the loss of your father. Would you like to join us in our journey? Your help would be useful to us."
```

Comment:
I think with orcish culture, it makes sense for Kapou'e to say 'me' here. He is the driving force in this campaign, and everything in the campaign (that I can recall) revolves around his leadership.

Reply:
Will revert. "Me" could have been a deliberate intention of the author. I ended up making this edit because the previous sentence said "join _us_".

Reply:
Sounds reasonable, but in that case I would have just phrased it as 'Your help would be useful.'

Reply:
Sorry I didn't make it clear, since you said you were going to revert the change. I would have truncated the sentence _if_ the intention was to follow the 'join us in our journey' phrase. I saw the 'useful to me' as distinct from that. I think how it originally was ('useful to me') works better.

Reply:
"I would have truncated the sentence if the intention was to follow the 'join us in our journey' phrase"
I did this in my latest commit.

>Grüü, many thanks for your help in this fight. I’m sorry for the loss of your father. Would you like to join us in our journey? Your help would be useful.

The above is okay right?

Reply:
That's my point - I didn't think it made sense to separate Grüü's service from Kapou'e specifically. Hence I thought it better to revert to its original state:
> Your help would be useful to me.

If, however, you really have your heart set on combining the 'join us' and 'useful to us', I'll defer to whoever gives the final approval, which I assume will be nemaara. (Doesn't look like the original author would still be around, the line hasn't been changed for 13 years.)

Reply:
Finalizing revert in https://github.com/wesnoth/wesnoth/pull/6166/commits/5b89e3b965f4a67ded583b2e8cdf61a6ebbb7e23